src/test/scala/nodes/images/ImageBenchMarkSuite.scala

```diff
@@ -0,0 +1,143 @@
+package nodes.images
+
+import breeze.linalg._
+import breeze.stats._
+import org.scalatest.FunSuite
+import pipelines.Logging
+import utils._
+import utils.TestUtils._
+
+import scala.util.Random
+
+class ImageBenchMarkSuite extends FunSuite with Logging {
```

Comment:
I don't think sbt/sbt test gets run anywhere, but i can comment out this benchmark. Its not relevant to our current experiments I think.